app/etc/pulp/server.yaml

```diff
@@ -37,10 +37,7 @@
 #
 # logging:
 #   loggers:
-#     django:
-#       level: INFO
-#     pulp.app:
-#       level: INFO
+#     '':
```

Comment:
@bmbouter Validated by www.yamllint.com/ codebeautify.org/yaml-validator beautifytools.com/yaml-validator.php and all says it's okay. About the ` 'handlers': ["stream"],` It was missing from it before so I also leave it out, but probably it should be there, I will add it.